mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -61,6 +63,23 @@
 warnings.simplefilter('always')
 
 
+def test_vmrk_meas_date():
+    """Test successful extraction of measurement date."""
+    # Test file that does have a specific date
+    raw = read_raw_brainvision(vhdr_path)
+    assert_allclose(raw.info['meas_date'], [1384359243, 794231])
+    assert '2013-11-13 16:14:03 GMT' in repr(raw.info)
+
+    # Test file with multiple dates ... we should only take the first
+    raw = read_raw_brainvision(vhdr_old_path)
+    assert_allclose(raw.info['meas_date'], [1184588560, 937453])
+    assert '2007-07-16 12:22:40 GMT' in repr(raw.info)
+
+    # Test files with no date, we should get DATE_NONE from mne.io.write
+    raw = read_raw_brainvision(vhdr_v2_path)
+    assert_allclose(raw.info['meas_date'], DATE_NONE)
```

Comment:
It would be nice to add back the `assert ` for this case but change `__repr__` so it handles `DATE_NONE` properly

Reply:
Okay, what should be inserted into the `repr` string instead of `1970-01-01 00:35:47 GMT`, once  `DATE_NONE` is encountered? something like "NO MEASUREMENT DATE RECORDED"?

Reply:
`meas_date : ndarray | Unknown date`?

Reply:
or `meas_date : ndarray | unspecified` or `unset` or ...